wagtailio/project_styleguide/templates/patterns/pages/showcase_page/showcase_page_fragment.html

```diff
@@ -0,0 +1,57 @@
+{% load wagtailcore_tags wagtailimages_tags %}
+<div id="showcase_results" class="showcase-grid">
+    {% for item in showcase_items %}
+        <div hx-on:click="window.location.replace('{{ item.url }}')" class="showcase-card">
+            {% if pattern_library %}
+                {# I've tried everything to get this to render using the pattern-library, #}
+                {# This is a placeholder until a fix is found. #}
+                <img src="https://placehold.co/600x400" width="600" height="400" alt="Some alt" class="showcase-card__image">
+                <img src="https://placehold.co/120x100" width="120" height="100" alt="Some alt" class="showcase-card__logo">
+            {% else %}
+                <picture>
+                    {% image item.image max-600x400 format-webp as webp_image %}
+                    <source srcset="{{ image_webp.url }}"
+                            type="image/webp"
+                            alt="{% firstof item.alt_text item.image.alt %}"/>
+                    {% image item.image max-600x400 as image_png %}
+                    <source srcset="{{ image_png.url }}"
+                            type="image/png"
+                            alt="{% firstof item.alt_text item.image.alt %}"/>
+                    {% image item.image max-600x400 as base_img %}
+                    <img
+                        src="{{ base_img.url }}"
+                        height="{{ base_img.height }}"
+                        width="{{ base_img.width }}"
+                        class="showcase-card__image"
+                        alt="{% firstof item.alt_text item.image.alt %}"/>
+                </picture>
+                {% if item.logo %}
+                    <picture>
+                        {% image item.logo max-120x100 format-webp as webp_image %}
+                        <source srcset="{{ image_webp.url }}"
+                                type="image/webp"
+                                alt="{% firstof item.alt_text item.image.alt %}"/>
+                        {% image item.logo max-120x100 as image_png %}
+                        <source srcset="{{ image_png.url }}"
+                                type="image/png"
+                                alt="{% firstof item.alt_text item.image.alt %}"/>
+                        {% image item.logo max-120x100 as base_img %}
+                        <img
+                            src="{{ base_img.url }}"
+                            height="{{ base_img.height }}"
+                            width="{{ base_img.width }}"
+                            class="showcase-card__logo"
+                            alt="{% firstof item.alt_text item.image.alt %}"/>
+                    </picture>
+                {% endif %}
+            {% endif %}
+            <div class="showcase-card__content">
+                <h2 class="showcase-card__heading">{{ item.title }}</h2>
+                <p class="showcase-card__description">{{ item.description }}</p>
```

Comment:
Currently the showcase item description can be empty. Is that intended? If so, do we need the empty `p.showcase-card__description` ?

Reply:
Yeah, that's a good spot. let's enforce it and keep it consistent in the UI.